modules/auth/proc-adding-google-as-an-authentication-provider.adoc

```diff
@@ -0,0 +1,59 @@
+[id='proc-adding-google-as-an-authentication-provider_{context}']
+= Enabling the Google authentication provider
+
+{product} includes a Google authentication provider that can authenticate users by using Google OAuth2.
+
+.Procedure
+. To allow {product-short} to authenticate with Google, create OAuth credentials in Google.
+For more information, see the official Google Cloud documentation for link:https://developers.google.com/identity/openid-connect/openid-connect#appsetup[Setting up OAuth 2.0].
++
+.. To authorize {product-short} to access the user consent screen, enter the necessary information on the https://console.cloud.google.com/apis/credentials/consent/edit[*Google Cloud > Oauth consent screen > Edit app registration*] page, including the following:
```

Comment:
```suggestion
.. To authorize {product-short} to access the user consent screen, enter the necessary information on the https://console.cloud.google.com/apis/credentials/consent/edit[*Edit app registration*] page, including the following:
```

Reply:
The name of the page is **Edit app registration**. Adding the nav path make the name page incorrect and is also unnecessary since we link directly to the page.